kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,14 @@
     });
   }
 
+  var draftbutton = document.getElementsByClassName('btn-draft')[0];
+  var url = $( '.btn-draft' ).data( 'draft-url' );
+  draftbutton.addEventListener('click', function() {
+    var both_data = $('#both_form').serializeArray();
+    var doc_data = $('#doc_form').serializeArray();
+    var rev_data = $('#rev_form').serializeArray();
+    var total_data = both_data.concat(doc_data, rev_data);
```

Comment:
Mixing Vanilla-JS DOM access and jQuery DOM access is confusing. You should pick one and use that for all the operations.